src/main/java/de/False/BuildersWand/events/WandStorageEvents.java

```diff
@@ -80,7 +81,7 @@ public void dropItem(PlayerDropItemEvent event)
         Player player = event.getPlayer();
         InventoryView openInventory = player.getOpenInventory();
         Inventory storage = openInventory.getTopInventory();
-        if(storage == null || !storage.getName().equals(INVENTORY_NAME))
+        if(storage == null || !(storage.getHolder() instanceof BWHolder))
```

Comment:
That's the recommended way to check Inventories.
Since 1.14, you even _have to_ use this way...